docs/modules/quickstarters/partials/secret-scanning-with-gitleaks.adoc

```diff
@@ -0,0 +1,13 @@
+== Secret scanning with gitleaks
+
+This repository has a pre-commit hook that runs gitleaks on every commit.
+To install the hook, run:
```

Comment:
I'd suggest to include a reference to the alternatives, e.g.
```suggestion
To https://pre-commit.com/#install[install the hook], run:
```
This would lower the barrier of actually using a pre-commit hook.

Reply:
Cloud quickstarters include Makefile to ease local setup.
Maybe we could add Makefile in any qs and just harmonize on `make init`.

https://github.com/opendevstack/ods-quickstarters/blob/master/inf-terraform-aws/files/Makefile#L36

Reply:
It doesn't have to be make itself  but the same approach, if task/Taskfile is better....

Though, not sure how many developers use Windows for all those QS environments and if those would not use WSL2 anyway.

Reply:
the Cloud QS is heavily using make and we confuse users when we introduce a different approach here.

Reply:
Other quickstarters are using different build tools, or is this PR somehow specific to the cloud qs?